docs/content/foundation/typography/paragraph-styles.mdx

```diff
@@ -4,10 +4,30 @@ nav_label: Paragraph Styles
 nav_order: 30
 ---
 
-# NCIDS Paragraph Styles
+# Paragraph Styles
 
-Nulla facilisis vitae lectus quis ultricies. Aliquam sodales scelerisque sapien et dictum.
-Mauris eu nunc sit amet lorem blandit gravida et ac dui. Suspendisse accumsan eros sit
-amet arcu lobortis, in consequat nunc molestie. Aenean lobortis molestie convallis.
-Nullam sit amet turpis vel nisi tristique dapibus sed in libero.
-Vivamus ac tristique nisl, eu bibendum eros.
+Change the appearance of a block of text.
+
+## Examples and usage
+
+Refer to [USWDS Paragraph Styles](https://designsystem.digital.gov/utilities/paragraph-styles/) for examples and usage on line height, measure, text alignment and text indent.
+
+## Custom NCIDS Settings
+
+Theme token updates:
+
+- `$theme-body-line-height: 5 !default;`
+- `$theme-text-measure: 6 !default;`
+- `$theme-text-measure-wide: 6 !default;`
+
+## Line Height Tokens
+
```

Comment:
🙏 can you add the line-hight table from this page to here https://designsystem-dev.cancer.gov/pr-1272/foundation/typography/fonts